### DISCUSSION TITLE
Is it possible to not show file name nor dir name?

### DISCUSSION THREAD

**Discussion by samodostal** [utilyre/barbecue.nvim#62] [Q&A] [answered]
I would like to show only the 'navic' bar, not file name or dir name. I didn't find an option for this, and couldn't figure it out the `modifiers.dirname` option

**Comment by utilyre** [comment]
Should be a simple if statement.

**Comment by utilyre** [comment, marked as answer]
Update to latest commit and set both `show_dirname` and `show_basename` to `false`.

**Reply by samodostal** [reply]
Thanks for the quick response!
